src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -516,6 +519,34 @@ def run(self, timeout):
       raise CorpusPruningException(
           'Corpus pruning failed to merge shared corpus\n' + repr(e))
 
+    return result.stats
+
+
+def record_cross_pollination_stats(
+    pruner_stats, pollinator_stats, project_qualified_name, sources, tags,
+    initial_corpus_size, minimized_corpus_size_units):
+  """Log stats about cross pollination in BigQuery."""
+  if pruner_stats and pollinator_stats and not environment.get_value(
```

Comment:
Nit: rather than having the whole function here indented, return in the opposite case.

I also generally find it easier to reason about positive rather than negative variable names (ENABLE vs DISABLE) but don't think it will matter too much in this case once the condition is flipped.